actions/update_travelnet.lua

```diff
@@ -73,6 +73,11 @@ return function (node_info, fields, player)
 	local network
 	local timestamp = os.time()
 	if owner_name ~= fields.owner_name then
+		if not minetest.check_player_privs(player_name, { travelnet_attach = true }) then
```

Comment:
```suggestion
		if not minetest.check_player_privs(player_name, { travelnet_attach = true })
			and not travelnet.allow_attach(player_name, owner_name, fields.station_network) then
```
I think this would work and allow it to do similar overridable checks that are done @ add action. Didn't test...